packages/commonwealth/client/scripts/views/components/component_kit/CWCoverImageUploader/CWCoverImageUploader.scss

```diff
@@ -0,0 +1,169 @@
+@import '../../../../../styles/shared.scss';
+
+.CWCoverImageUploader {
+  display: flex;
+  flex-direction: column;
+  gap: 2px;
+  height: 100%;
+  width: 100%;
+
+  .UploadControl {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 16px;
+    height: 100%;
+    width: 100%;
+    padding: 16px;
+    background-color: $neutral-25;
+    border: 1px solid $neutral-200;
+    border-radius: $border-radius-corners;
+    position: relative;
+    cursor: pointer;
+    min-height: 175px;
+
+    &.hasImageURL {
+      min-height: 270px;
+    }
+
+    &.isLoading {
+      cursor: default;
+    }
+
+    &.disabled {
+      cursor: not-allowed;
+    }
+
+    &.hovered {
+      border: 1px solid $primary-500;
+    }
+
+    .file-input {
+      display: none;
+    }
+
+    .img-circle {
+      height: 180px;
+      width: 180px;
+      border-radius: 50%;
+      object-fit: cover;
+    }
+
+    .img-cover {
+      width: 100%;
+      max-height: 100%;
+      object-fit: cover;
+      position: absolute;
+      top: 0;
+      left: 0;
+      border-radius: $border-radius-corners;
+    }
+
+    // implement this when/if needed
```

Comment:
I would remove it

Reply:
I will actually be adding this now, based on `3` above. Ty for spotting that!